src/components/pages/ParentNewsFeed/index.js

```diff
@@ -4,21 +4,20 @@ import '../../../styles/index.less';
 import NewsContainer from './NewsContainer';
 import ParentSidebar from '../ParentHome/ParentSidebar';
 import Banner from '../../common/Banner';
-import TitleParentNewsFeed from './TitleParentNewsFeed';
+// import TitleParentNewsFeed from './TitleParentNewsFeed';
+import IndividualNewsParent from './IndividualNewsParent';
 const { Content } = Layout;
 
 function ParentNewsFeed() {
   return (
-    <div className="news-feed-page">
-      <Layout>
-        <ParentSidebar active="newsfeed" />
-        <Content>
-          <Banner />
-          <TitleParentNewsFeed />
-          <NewsContainer />
-        </Content>
-      </Layout>
-    </div>
+    <Layout className="news-feed-page">
+      <ParentSidebar active="newsfeed" />
+      <Content>
+        <Banner />
+        <IndividualNewsParent />
+        {/* <NewsContainer /> */}
```

Comment:
Same thing also, did you leave this commented out for a reason?

Reply:
@magoha01 that's a good question for hailey haha

Reply:
@JessWillCode should be gone when my PR is merged, if it isn't already!